README.md

```diff
@@ -118,8 +128,21 @@ Further, if there are any formula fields that are a third degree referential for
 * `key` (required): The key column within `fivetran_formula` you are querying. This argument is typically `field`.
 * `value` (required): The value column within `fivetran_formula` you are querying. This argument is typically `sql`.
 * `join_to_table` (required): The table with which you are joining the formula fields.
+* `added_inclusion_fields` (optional, default is none): The list of fields you want to be included in the macro. If no fields are selected then all fields will be included.
+* `not_null_value` (optional, default is true): Used by the macro to identify if the `null` fields within the `view_sql` column should be included.
```

Comment:
does this actually ever get used? we set `not_null_value` to true in the macros

Reply:
or is this supposed to be `no_nulls`?

Reply:
Good catch this should be `no_nulls`

Reply:
Updated